plugin/core/registry.py

```diff
@@ -144,11 +144,33 @@ def client_with_capability(self, capability: str) -> Optional[Client]:
         return client_from_session(session_for_view(self.view, capability))
 
 
-class LspRestartClientCommand(sublime_plugin.TextCommand):
+class LspRestartServersCommand(sublime_plugin.TextCommand):
```

Comment:
Since this is gonna to be a BC anyway. I would like to propose naming it as `LspRestartAllServersCommand`.

Reply:
I agree, I think that would make more sense